backends/text-to-speech/linux/linux-text-to-speech.h

```diff
@@ -59,7 +60,8 @@ class SpeechDispatcherManager : public Common::TextToSpeechManager {
 	SpeechDispatcherManager();
 	virtual ~SpeechDispatcherManager() override;
 
-	virtual bool say(Common::String str, Action action, Common::String charset = "") override;
+	virtual bool say(const Common::U32String &str, Action action, Common::String charset = "") override;
```

Comment:
What is the charset argument for here? Surely the string is in UTF-32 since it is passed as a `Common::U32String`?